sources/packages/backend/apps/queue-consumers/src/processors/schedulers/student-application-notifications/student-application-notifications.scheduler.ts

```diff
@@ -39,7 +44,28 @@ export class StudentApplicationNotificationsScheduler extends BaseScheduler<void
         `Processing student application notifications job. Job id: ${job.id} and Job name: ${job.name}.`,
       );
 
-      // TODO: Get applications that have a disability status mismatch  and check PDPPD status
+      const eligibleApplications =
+        await this.applicationService.getEligibleApplicationsForNotification();
+
+      for (const application of eligibleApplications) {
+        const notification: StudentPDPPDNotification = {
+          userId: application.student.user.id,
+          givenNames: application.student.user.firstName,
+          lastName: application.student.user.lastName,
+          email: application.student.user.email,
+          applicationNumber: application.applicationNumber,
+        };
```

Comment:
The `for` can be converted to a `map` as below and the method `saveStudentApplicationPDPPDNotification` should be changed to receive an `StudentPDPPDNotificationp[]` (follow the same from the [saveStudentRestrictionAddedNotification](https://github.com/bcgov/SIMS/blob/4fb892f1dccfa1243904d3521fd410ab939b6d71/sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts#L301)).
```ts
const notifications = eligibleApplications.map<StudentPDPPDNotification>(
  (application) => ({
    userId: application.student.user.id,
    givenNames: application.student.user.firstName,
    lastName: application.student.user.lastName,
    email: application.student.user.email,
    applicationNumber: application.applicationNumber,
    assessmentId: application.currentAssessment.id
  }),
);
```